packages/plugin-doc-coverage/src/lib/config.unit.test.ts

```diff
@@ -0,0 +1,112 @@
+import { describe, expect, it } from 'vitest';
+import {
+  type DocCoveragePluginConfig,
+  docCoveragePluginConfigSchema,
+} from './config.js';
+
+describe('docCoveragePluginConfigSchema', () => {
+  it('throws when skipAudits and onlyAudits are defined', () => {
+    expect(() =>
+      docCoveragePluginConfigSchema.parse({
+        skipAudits: ['functions-coverage'],
+        onlyAudits: ['classes-coverage'],
+      }),
+    ).toThrow("You can't define 'skipAudits' and 'onlyAudits' simultaneously");
+  });
+
+  describe('sourceGlob', () => {
```

Comment:
Let's use nesting of describe blocks only to encapsulate setup logic not to structure the tests.
This makes is reasier to read and maintain.

We can move the text from the describe into the it block

Reply:
I agree